internal/internal.go

```diff
@@ -519,6 +521,22 @@ func (p *envoyExtAuthzGrpcServer) log(ctx context.Context, input interface{}, re
 		info.Path = p.cfg.Path
 	}
 
+	sctx := trace.SpanFromContext(ctx).SpanContext()
+	if sctx.IsValid() {
+		info.TraceID = sctx.TraceID().String()
+		info.SpanID = sctx.SpanID().String()
+	}
+
+	p.manager.Logger().WithFields(map[string]interface{}{
```

Comment:
We should remove this log as much of this is already logged at debug level in the `check` call. This is also resulting in the test failure.